.kokoro/run_integration_tests.sh

```diff
@@ -10,7 +10,7 @@ cd $SCRIPT_DIR
 cd ..
 
 # We need to spawn and kill process and need to be root for this to work.
-sudo su
+sudo -E su <<EOF
```

Comment:
What does EOF do in this scenario?

Reply:
It's a delimiter once it hits the second one it will stop reading the input (which is the middle section here)